utils/gyb.py

```diff
@@ -1035,7 +1035,9 @@ def execute_template(
 
     Keyword arguments become local variable bindings in the execution context
 
-    >>> ast = parse_template('/dummy.file', text=
+    >>> root_directory = os.path.abspath('/')
```

Comment:
This is the best I can do. Basically `os.path.abspath('/')` == `/` or `C:/` depending on the platform.

Since doctest doesn't really handle platform specific tests, we then replace the full absolute file path to abstract away the platform specific code.